pkg/crc/preflight/preflight_darwin.go

```diff
@@ -117,6 +127,12 @@ func RegisterSettings() {
 }
 
 func CleanUpHost() {
-	logging.Warn("Cleanup is not supported for MacOS")
-	doCleanUpPreflightChecks(getPreflightChecks())
+	// A user can use setup with experiment flag
```

Comment:
the flag is named 'experimental'.
I guess we could name it in full `with the --enable-experimental-features` flag